src/v/kafka/client/record_batcher.cc

```diff
@@ -47,7 +47,7 @@ class batcher_impl {
         size_t record_size = record.size_bytes();
         if (record_size > max_records_bytes()) {
             vlog(
-              tlg_log.info,
+              kclog.info,
```

Comment:
comment/question: this I think means that if the transform logging system appends a record that is larger than the maximum record size, it will be the `kafka/client` logger facility that will report this?  Is this desirable?

Reply:
yeah good point. probably better to inject a logger object I guess, particularly since these are transient objects.

Reply:
>inject a logger

that's done. lmk what you think